test/test.highlevelConsumer.js

```diff
@@ -0,0 +1,222 @@
+'use strict';
+var util = require('util');
+var EventEmitter = require('events');
+var sinon = require('sinon');
+
+var libPath = process.env['KAFKA_COV'] ? '../lib-cov/' : '../lib/';
+
+// var Client = require(libPath + 'client');
```

Comment:
I think this was copied from another test. This comment should be deleted. And on the line below I am not sure what this `KAFKA_COV` was used for but we should delete that as well.